app/views/hud_reports/index.haml

```diff
@@ -1,6 +1,6 @@
 %h1 HUD Reports
 
 - reports = Rails.application.config.hud_reports.values.uniq
-- reports.each do |r|
+- reports.sort_by {|r| r[:title] }.each do |r|
```

Comment:
Maybe we don't want this but should we sort the hud reports alphabetically?

Reply:
This is good, thanks.  I don't think the current sort order was intentional, and once you are in one, the side-menu is sorted alphabetically.